wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.swift

```diff
@@ -38,6 +38,35 @@ final class ConversationContentViewController: UIViewController, PopoverPresente
         }
     }
 
+    let scrollToBottomButtonTrailingMargin: CGFloat = -10
+    let scrollToBottomButtonBottomMargin: CGFloat = -10
```

Comment:
I would keep the value as 10.

Reply:
for bottom and trailing needs to be -

Reply:
we usually add - when adding constraints

Reply:
what about having only one value: `scrollToBottomButtonMargin`?  since they are the same.

Reply:
> we usually add - when adding constraints

@KaterinaWire Ahhh, that's what you meant! I can make the change I guess. Good point! 👍

Reply:
> what about having only one value: `scrollToBottomButtonMargin`? since they are the same.

Do you think it's going to be clear enough, since it's used on two different constraints?

Reply:
> > we usually add - when adding constraints
> 
> @KaterinaWire Ahhh, that's what you meant! I can make the change I guess. Good point! 👍

Done!